Makefile

```diff
@@ -7,7 +7,14 @@ MSCS_INIT_D := /etc/init.d/mscs
 MSCS_SERVICE := /etc/systemd/system/mscs.service
 MSCS_COMPLETION := /etc/bash_completion.d/mscs
 
-.PHONY: install update clean
+.PHONY: docker-build docker-run install update clean
+
+docker-build:
```

Comment:
Since you placed docker-build and docker-run first in the list, it is now the default rather than install.  I'm not sure everyone wants to use docker.